### ISSUE TITLE
Broken Links

### ISSUE THREAD

**Issue by AnalogMan151** [Plailect/Guide#666]
Here is a list of pages that link to 404 pages I have found:

https://plailect.github.io/Guide/homebrew-launcher-(browser) -> https://plailect.github.io/Guide/9.2.0-Downgrade (404)

https://plailect.github.io/Guide/decrypt9-(homebrew-launcher) -> https://plailect.github.io/Guide/9.2.0-Downgrade (404)

https://plailect.github.io/Guide/homebrew-launcher-(no-browser) ->
https://plailect.github.io/Guide/9.2.0-Downgrade (404)

Basically, either update these links to point to https://plailect.github.io/Guide/9.2.0-downgrade or rename https://plailect.github.io/Guide/9.2.0-downgrade to https://plailect.github.io/Guide/9.2.0-Downgrade

**Comment by Plailect** [comment]
https://github.com/Plailect/Guide/commit/a219309b7a1c879f4f34a0314e76251abb96e7f2
